### DISCUSSION TITLE
Init a new db on external PG server

### DISCUSSION THREAD

**Discussion by emoriver** [immich-app/immich#8586] [Q&A]
I'm trying to install Immich with Docker and a I already have an external PG 15 installed (I maneged to install vecto.rs without troubles): to obtain that I commented out in docker compose file every reference to 'database' and on the .env file i point to the external IP and I changed accordingly user/pw. When I start the containers in the log of Immich Server I read something like 'Unable to find "immich" database..." which is not so strange because nothing/nobody created it... Where can I find a init query, something like that to create the database?

**Comment by bo0tzz** [comment]
Creating the database is something you would handle yourself before starting up Immich.

**Reply by emoriver** [reply]
Great... but I can't find any query or procedure to do so by myself

**Reply by bo0tzz** [reply]
https://www.postgresql.org/docs/current/manage-ag-createdb.html

**Reply by emoriver** [reply]
Ah ok! I know well ho to create a new db, you're telling me that once the schema is done the procedure will create tables etc etc... I try! Thanks

**Reply by emoriver** [reply]
It worked! It was easy... thanks!
